api_documentation/api-gateway/index.md

```diff
@@ -0,0 +1,11 @@
+API Gateway
+===========
+
+This is a microservice that sits in front of most SMARTA srevices, although it can also be accessed independently at `api-gateway.services.smartatransit.com` for debugging purposes. It serves to validate and parse SMARTA API authorization and pass it on to the internal server. It behaves as follows:
```

Comment:
```suggestion
This is a microservice that sits in front of most SMARTA services, although it can also be accessed independently at `api-gateway.services.smartatransit.com` for debugging purposes. It serves to validate and parse SMARTA API authorization and pass it on to the internal server. It behaves as follows:
```